interop/src/main.rs

```diff
@@ -257,19 +257,23 @@ impl State {
             .to_socket_addrs()?
             .next()
             .ok_or_else(|| anyhow!("couldn't resolve to an address"))?;
-        let host = if webpki::DNSNameRef::try_from_ascii_str(&peer.host).is_ok() {
+        let host = if webpki::DnsNameRef::try_from_ascii_str(&peer.host).is_ok() {
             &peer.host
         } else {
             warn!("invalid hostname, using \"example.com\"");
             "example.com"
         };
 
-        let mut tls_config = rustls::ClientConfig::new();
-        tls_config.versions = vec![rustls::ProtocolVersion::TLSv1_3];
+        let verifier = InteropVerifier(Arc::new(Mutex::new(false)));
+        let mut tls_config = rustls::ClientConfig::builder()
+            .with_safe_default_cipher_suites()
+            .with_safe_default_kx_groups()
+            .with_protocol_versions(&[&rustls::version::TLS13])
+            .unwrap()
+            .with_custom_certificate_verifier(Arc::new(verifier))
+            .with_no_client_auth();
+
         tls_config.enable_early_data = true;
```

Comment:
No setter on the builder for that and alpn?

Reply:
Nope.